Web-terminal/README.md

```diff
@@ -0,0 +1,13 @@
+# Web-Terminal
+
```

Comment:
Please add a screenshot of the snippet in action.

Reply:
![image](https://user-images.githubusercontent.com/115607723/195358706-596932aa-3090-4c05-bdb9-707901b00813.png)

Reply:
Yes please add the Screenshot to the Readme!

Reply:
ok wait

Reply:
Ok check now

Reply:
Nope, add the screenshot to the readme!

Reply:
Sorry, just noticed that you have added it!